extensions/ui/org.eclipse.smarthome.ui.paper/web-src/partials/control.html

```diff
@@ -56,14 +56,6 @@ <h3 class="clabel">{{thing.label}}</h3>
 													</div>
 												</div>
 												<md-slider class="md-default-theme" style="height: 40px;" flex min="0" max="100" ng-model="item.state" ng-change="setBrightness(item.state)" aria-label={{item.label}}></md-slider>
-												<div layout="row" layout-align="start center" class="dimmer-widget" ng-if="!isHideSwitch(item.category)">
```

Comment:
I only noticed now that there is already a feature "isHideSwitch" implemented, which allows hiding the toggle based on the category - imho it makes sense to keep this feature in place.

I would only suggest to change the default: If there is no exception defined in the category list, the toggle should be hidden. Only if a "showSwitch=true" is defined, it should be shown. For now, I would show it for DimmableLights only.